docs/api/ref/DesktopAgent.md

```diff
@@ -498,7 +521,7 @@ joinUserChannel(channelId: string) : Promise<void>;
 
 Optional function that joins the app to the specified User channel. In most cases, applications SHOULD be joined to channels via UX provided to the application by the desktop agent, rather than calling this function directly.
```

Comment:
```suggestion
Optional function that joins the app to the specified User channel. In most cases, applications SHOULD be joined to channels via UX provided to the application by the Desktop Agent, rather than calling this function directly.
```